dependencies.sh

```diff
@@ -39,4 +39,4 @@ systemctl enable osbuild-composer.socket cockpit.socket --now
 
 cat requirements.txt  | xargs -n1 $PYTHON_CMD -m pip install
 
-sudo activate-global-python-argcomplete
+sudo $PYTHON_CMD $(which activate-global-python-argcomplete)
```

Comment:
@konguyen @bn222  I think this might have broken CI. We are running into the same issue when running manually too

```
Traceback (most recent call last):
  File "/tmp/cda-venv/bin/activate-global-python-argcomplete", line 5, in <module>
    from argcomplete.scripts.activate_global_python_argcomplete import main
ModuleNotFoundError: No module named 'argcomplete'
make: *** [Makefile:85: prepare-e2e-test] Error 1
```
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_dpu-operator/348/pull-ci-openshift-dpu-operator-main-make-e2e-test/1905694165056884736

Reply:
Here is the root cause
```
(ocp-venv) [root@prov]# which $PYTHON_CMD
/root/dpu-operator/cluster-deployment-automation/ocp-venv/bin/python
(ocp-venv) [root@prov]# sudo -E which $PYTHON_CMD
/bin/python
```
Maybe what you should rather do is the below
```
export PYTHON_CMD_FULL=$(which $PYTHON_CMD)
sudo $PYTHON_CMD_FULL $(which activate-global-python-argcomplete)
```

Reply:
https://github.com/bn222/cluster-deployment-automation/pull/328